lib/matplotlib/rcsetup.py

```diff
@@ -1389,6 +1389,7 @@ def _validate_linestyle(ls):
     'keymap.yscale':       [['l'], validate_stringlist],
     'keymap.xscale':       [['k', 'L'], validate_stringlist],
     'keymap.all_axes':     [['a'], validate_stringlist],
+    'keymap.copy':         [('ctrl+c', 'cmd+c', 'shift+ctrl+c'), validate_stringlist],
```

Comment:
list instead of tuple? (as above, basically)

Reply:
@anntzer  Done